pkg/restore/restore.go

```diff
@@ -76,31 +76,31 @@ type kubernetesRestorer struct {
 // prioritizeResources takes a list of pre-prioritized resources and a full list of resources to restore,
 // and returns an ordered list of GroupResource-resolved resources in the order that they should be
 // restored.
-func prioritizeResources(helper discovery.Helper, priorities []string, includedResources *collections.IncludesExcludes, logger *logrus.Logger) ([]schema.GroupResource, error) {
+func prioritizeResources(helper discovery.Helper, priorities []string, resources []*metav1.APIResourceList, includedResources *collections.IncludesExcludes, logger *logrus.Logger) ([]schema.GroupResource, error) {
 	var ret []schema.GroupResource
 
 	// set keeps track of resolved GroupResource names
 	set := sets.NewString()
 
 	// start by resolving priorities into GroupResources and adding them to ret
 	for _, r := range priorities {
-		gr, err := helper.ResolveGroupResource(r)
+		gvr, _, err := helper.ResourceFor(schema.ParseGroupResource(r).WithVersion(""))
 		if err != nil {
 			return nil, err
 		}
+		gr := gvr.GroupResource()
 
 		if !includedResources.ShouldInclude(gr.String()) {
 			logger.WithField("groupResource", gr).Info("Not including resource")
 			continue
 		}
-
 		ret = append(ret, gr)
 		set.Insert(gr.String())
 	}
 
 	// go through everything we got from discovery and add anything not in "set" to byName
 	var byName []schema.GroupResource
-	for _, resourceGroup := range helper.Resources() {
+	for _, resourceGroup := range resources {
```

Comment:
Was there a reason for this change?

Reply:
good q, not really/may have been a remnant of some rebasing from before. fixed it.